### DISCUSSION TITLE
Tailwind CSS in motion: Show us what you built with animation/transition utilities! 👀

### DISCUSSION THREAD

**Discussion by simonswiss** [tailwindlabs/tailwindcss#2463] [Show and tell]
Hey folks! 👋

Tailwind CSS has recently added [animations](https://www.youtube.com/watch?v=75_a2lTJSjA&) and [transitions](https://www.youtube.com/watch?v=AYoQfPX31Mg) utilities to the framework.

Would love to see what y'all have been building with them, so let's start a **Tailwind CSS in motion** showcase thread! 🎉

**Comment by ecklf** [comment]
Nothing super special, but I used `.animate-ping` in a kind of *unusual* way. It marks a game objective in a three.js scene.

![ezgif-6-269ea3c4840f](https://user-images.githubusercontent.com/8146736/94549950-45b59000-0253-11eb-93c9-ce7669bd0dda.gif)

**Reply by simonswiss** [reply]
Wow - definitely _unusual_, but works great, love it! 

Good stuff, thanks for sharing 🎉
